edxadapt/static/html/student_view.html

```diff
@@ -3,16 +3,15 @@
     data-student-id="{anonymous_student_id}"
     data-api-base-url="{edx_adapt_api_url}"
     data-course-id="{course_id}"
-    data-params="{params}"
-    data-skills="{skills}"
+    data-params={params}
```

Comment:
So will python's `str.format()` surround list and dict with quotes and this become a valid html?